log.go

```diff
@@ -0,0 +1,16 @@
+package migrator
+
+import "log"
+
+var (
+	Debug  = false
```

Comment:
Global variables are not a good programming practice, so I would not agree to keep them in the codebase.

If you wanna do it the proper way, it should be set as a property of `migrator.Migrator{}`

Reply:
The first idea is the same, but the place where sql is executed later is implemented by function transfer, which will be passed multiple times, and the test case is also used, and the changes are relatively large, so it is forced to use the global variable method

Reply:
It is still not a good reason to use the global variable 🙂 Better explicit definition in many places instead of global scope. It does not allow you to test it and you never know how it might affect your functionality